tracer/src/Datadog.Trace/Telemetry/Metrics/IntegrationIdExtensions.cs

```diff
@@ -75,7 +75,7 @@ public static MetricTags.IntegrationName GetMetricTag(this IntegrationId integra
             IntegrationId.TestPlatformAssemblyResolver => MetricTags.IntegrationName.TestPlatformAssemblyResolver,
             IntegrationId.StackTraceLeak => MetricTags.IntegrationName.StackTraceLeak,
             IntegrationId.XpathInjection => MetricTags.IntegrationName.XpathInjection,
-            IntegrationId.ReflectionInjection => MetricTags.IntegrationName.ReflectionInjection,
```

Comment:
Why is this one gone?

Reply:
Merge conflict bug 😬 Will fix in redirection after rebase

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/5235/commits/cf170ae0d62456a036d9d263240d9b1edf93ac5e